ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -355,7 +355,7 @@ open class SegmentedControl: UIView, TokenizedControlInternal {
     open override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
         invalidateIntrinsicContentSize()
-        layoutSubviews()
+        setNeedsLayout()
```

Comment:
(1) in the layoutSubview function do we still need pillContainerView.flipSubviewsForRTL() and flipSubviewsForRTL() ? most of views are in uistackview which automatically flips. only thing that it might need fliping is pillMaskedContentContainerView?  (2) it seems like your UIhierarchy comments might be outdated?

Reply:
because afaik "pillMasked**Labels**ContainerView " doesn't exist?